src/legacy/ui/public/agg_types/metrics/median.js

```diff
@@ -37,7 +37,7 @@ export const medianMetricAgg = new MetricAggType({
     {
       name: 'field',
       type: 'field',
-      filterFieldTypes: 'number'
+      filterFieldTypes: 'number, date'
```

Comment:
Could you please use the array syntax for that instead:

```suggestion
      filterFieldTypes: ['number', 'date']
```